playground/infrastructure/ci_helper.py

```diff
@@ -0,0 +1,55 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from api.v1.api_pb2 import SDK_JAVA
+from helper import Example, get_statuses
+
+SUPPORTED_SDK = {'java': SDK_JAVA}
```

Comment:
Could you move it into constants? You can use dataclass with frozen=true to make it truly const.

Reply:
Changed.

Reply:
Maybe move constants to config.py?